resources/overrides/library/index.d.ts

```diff
@@ -0,0 +1,33 @@
+import "./sap.ca.ui";
+import "./sap.chart";
+import "./sap.collaboration";
+import "./sap.f";
+import "./sap.gantt";
+import "./sap.m";
+import "./sap.makit";
+import "./sap.me";
+import "./sap.rules.ui";
+import "./sap.suite.ui.commons";
+import "./sap.suite.ui.generic.template";
+import "./sap.suite.ui.microchart";
+import "./sap.tnt";
+import "./sap.ui.commons";
+import "./sap.ui.comp";
+import "./sap.ui.core";
+import "./sap.ui.export";
+import "./sap.ui.generic.app";
+import "./sap.ui.integration";
+import "./sap.ui.layout";
+import "./sap.ui.mdc";
+import "./sap.ui.richtexteditor";
+import "./sap.ui.suite";
+import "./sap.ui.support";
+import "./sap.ui.table";
+import "./sap.ui.unified";
+import "./sap.ui.ux3";
+import "./sap.ui.vbm";
+import "./sap.ui.webc.fiori";
+import "./sap.ui.webc.main";
+import "./sap.ushell";
+import "./sap.uxap";
+import "./sap.viz";
```

Comment:
add last empty line

Reply:
It's autogenerated, but I'll try to do it 👍